README.md

```diff
@@ -505,17 +503,17 @@ Please see our [Upgrade Guide](./UPGRADING.md) for specific steps.
 
 ### Compatibility with OTel packages
 
-The SDK is built on top of OpenTelemetry and as such it is possible to use it alongside other OTel libraries. **Important:** The Embrace Web SDK only supports OpenTelemetry 1.x packages. OpenTelemetry 2.x (and above) is **not supported** and will not work with this SDK at this time.
+The SDK is built on top of OpenTelemetry and, as such, it is possible to use it alongside other OTel libraries. **Important: New projects should use OpenTelemetry 2.x.**
+OpenTelemetry 1.x support is limited to 1.x versions of the SDK, which are deprecated.
```

Comment:
I used the word `deprecated` here as a strong recommendation even though it's not actually deprecated on npm.